Technical/Logging/The-Tazama-Logging-Framework.md

```diff
@@ -1 +1,405 @@
 # The Tazama Logging Framework
+
+- [The Tazama Logging Framework](#the-tazama-logging-framework)
+   * [Logging](#logging)
+      + [Architecture](#architecture)
+         - [Sequence Diagram](#sequence-diagram)
+         - [Logger](#logger)
+         - [Event-Sidecar](#event-sidecar)
+            * [Server](#server)
+            * [Client](#client)
+         - [Lumberjack](#lumberjack)
+      + [Implementation](#implementation)
+         - [Configuration](#configuration)
+            * [With Event-Sidecar and Lumberjack](#with-event-sidecar-and-lumberjack)
+               + [Lumberjack](#lumberjack-1)
+               + [Event-Sidecar](#event-sidecar-1)
+               + [Processor](#processor)
+            * [Without Event-Sidecar and Lumberjack](#without-event-sidecar-and-lumberjack)
+         - [Usage](#usage)
+            * [Logger Service ](#logger-service)
+               + [Function Signature:](#function-signature)
+            * [Usage:](#usage-1)
+            * [Examples:](#examples)
+         - [Sequence Diagram:](#sequence-diagram-1)
+         - [Notes:](#notes)
+   * [APM](#apm)
+      + [Overview](#overview)
+      + [Compatibility](#compatibility)
+      + [Setup Instructions](#setup-instructions)
+      + [Features and Benefits](#features-and-benefits)
+      + [Usage](#usage-2)
+         - [Initialization](#initialization)
+            * [Configuration Options](#configuration-options)
+      + [Example Usage](#example-usage)
+      + [Troubleshooting](#troubleshooting)
+      + [Additional Resources](#additional-resources)
+
+
+## Logging
+### Architecture
```

Comment:
Please also provide a "context"/block diagram for the architecture that shows all the components and follow the diagram with a description/summary of each component and its purpose.

e.g. (I have no idea if this is accurate...)
![image](https://github.com/frmscoe/docs/assets/123470803/9095d0f3-2605-4e4b-bc1f-88b0b97a2590)